### ISSUE TITLE
Project setup instructions

### ISSUE THREAD

**Issue by verquist** [citizensgov/citizensgov2020#5]
We need to define the steps necessary to get our application up and running, and document this in our README. 

Lance initially had an issue creating the databases locally, but he got it working within a few minutes. I'm not sure what he did. I'm having trouble getting Rails to create the database on Heroku.

**Comment by verquist** [comment]
Yes, like you said: once you have Postgres installed, you can rake db:create and rake db:migrate. Or you can do a simple: rake db:setup. I tried this out on a separate, newly created rails app.

On Heroku I believe I need to provision the Postgres database first. This should be simple. I just haven't had a chance to do it yet.

**Comment by verquist** [comment]
Hey Dan, the instructions look great. I love the code formatting. I need to brush up on markdown. I apologize for closing the issue while you were working on it. I'll need to take out my redundant instructions.

The only discrepancy I noticed is that I used: rake db:setup. You used: rails db:setup. I assume you tested this and it works.

Are you up and running locally?

**Comment by DanGould** [comment]
@verquist check this out: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet. When I'm back on campus I can give you a little paper guide. If you're there, there should be print versions of this in Spark!

I have tested this. After rails 5 everything done with `rake` can be done with  `rails`. `rails db:setup` creates citizensgov2020_test & citizensgov2020_development dbs

I am up and running
